### ISSUE TITLE
Error while using YouTube plugin

### ISSUE THREAD

**Issue by arnxv0** [Flow-Launcher/Flow.Launcher#1050]
Flow Launcher version: 1.9.2
OS Version: Microsoft Windows NT 10.0.19044.0
IntPtr Length: 8
x64: True
Python Path: D:\arnav\Python\sdks\Python38

```
Date: 03/01/2022 20:07:35
Exception:
System.IO.InvalidDataException: FlowYouTube|Exception ignored in: <function Launcher.__del__ at 0x000001FD84BB9790>
Traceback (most recent call last):
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\lib\flox\launcher.py", line 59, in __del__
    self.__call__()
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\lib\flox\__init__.py", line 96, in __call__
    return super().__call__()
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\lib\flox\launcher.py", line 41, in __call__
    results = request_method(*request_parameters)
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\lib\flox\__init__.py", line 127, in _query
    self.query(query)
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\plugin\main.py", line 27, in query
    self._results = utils.cache(query, max_age=MAX_CACHE_AGE, dir=path)(self.search)(query)
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\lib\flox\utils.py", line 39, in wrapper
    data = func(*args, **kwargs)
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\plugin\main.py", line 35, in search
    self.result(item, executor)
  File "D:\arnav\FlowLauncher\data\FlowLauncher\app-1.9.2\UserData\Plugins\FlowYouTube-0.2.0\plugin\main.py", line 41, in result
    download_thumbs = strtobool(self.settings.get('download_thumbs', True))
  File "D:\arnav\Python\sdks\Python38\lib\site-packages\setuptools\_distutils\util.py", line 384, in strtobool
    val = val.lower()
AttributeError: 'bool' object has no attribute 'lower'

   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.ExecuteAsync(ProcessStartInfo startInfo, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 308
   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.ExecuteAsync(ProcessStartInfo startInfo, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 308
   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.QueryAsync(Query query, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 323
   at Flow.Launcher.Core.Plugin.PluginManager.<>c__DisplayClass22_0.<<QueryForPluginAsync>b__0>d.MoveNext() in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 192
--- End of stack trace from previous location ---
   at Flow.Launcher.Infrastructure.Stopwatch.DebugAsync(String message, Func`1 action) in C:\projects\flow-launcher\Flow.Launcher.Infrastructure\Stopwatch.cs:line 40
   at Flow.Launcher.Core.Plugin.PluginManager.QueryForPluginAsync(PluginPair pair, Query query, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 211
   at Flow.Launcher.ViewModel.MainViewModel.<>c__DisplayClass137_0.<<QueryResults>g__QueryTask|2>d.MoveNext() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 623
--- End of stack trace from previous location ---
   at Flow.Launcher.ViewModel.MainViewModel.QueryResults() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 624
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by Garulf** [comment]
Fixed in version v0.2.1 https://github.com/Garulf/FlowYouTube/releases/tag/v0.2.1

You'll be able to update when the store detects the update.

**Comment by arnxv0** [comment]
> Fixed in version v0.2.1 https://github.com/Garulf/FlowYouTube/releases/tag/v0.2.1
> 
> You'll be able to update when the store detects the update.

Thanks!
